scripts/GetClrDbg.ps1

```diff
@@ -104,13 +104,20 @@ if (-not $RuntimeID) {
 }
 Write-Host "Info: Using Runtime ID '$RuntimeID'"
 
-# create the install folder if it does not exist
-if (-not (Test-Path -Path $InstallPath -PathType Container)) {
-    New-Item -ItemType Directory -Force -Path $InstallPath
+# create the temp folder if it does not exist
+$GuidString = [System.Guid]::NewGuid()
+$TempPath = Join-Path -Path $env:TEMP -ChildPath $GuidString
```

Comment:
If you are going to do it this way (which sounds reasonable to me) you should probably also delete the directory at the end of the script.